src/test/java/org/broadinstitute/hellbender/engine/AssemblyRegionUnitTest.java

```diff
@@ -122,6 +124,19 @@ private void assertGoodReferenceGetter(final byte[] actualBytes, final SimpleInt
         return tests.subList(2,3).toArray(new Object[][]{});       //HACK!
     }
 
+    @Test(expectedExceptions = IllegalArgumentException.class, expectedExceptionsMessageRegExp = "Contig chrNotInReference not found in reference.*")
+    //Testing a case where an NPE was thrown when somehow the reference sequence dictionary didn't contain a contig in the
+    //assembly region.
+    public void testMimatchedReferenceAndRegion(){
+        final SAMSequenceDictionary sequenceDictionary = header.getSequenceDictionary();
+        final String contigNotInReference = "chrNotInReference";
+        sequenceDictionary.addSequence(new SAMSequenceRecord(contigNotInReference, 1000));
```

Comment:
Won't this change the sequence dictionary in the shared `header` instance used by all the tests in this class? Can you write this test in a way that doesn't modify shared data?

Reply:
Ugh. Yes it does. Changing.  Good catch.